unit/analyses/call_graph.cpp

```diff
@@ -223,6 +223,48 @@ SCENARIO("call_graph",
         REQUIRE(exported.has_edge(nodes_by_name["B"], nodes_by_name["D"]));
       }
 
+      THEN("We expect {A,B} to be reachable from {A} in 1 step")
+      {
+        irep_idt function_name = "A";
+        std::size_t depth = 1;
+        std::set<irep_idt> reachable = get_functions_reachable_within_n_steps(
+          exported, function_name, depth, true);
+        REQUIRE(reachable.size() == 2);
```

Comment:
Could use `REQUIRE(reachable == std::set<irep_idt> {"A", "B"});` and similar

Reply:
This check is the same code as is used in several other unit tests in this file, is there a reason mine should be different? Unit tests that use this are:
get_callees: https://github.com/diffblue/cbmc/blob/develop/unit/analyses/call_graph.cpp#L229-L231
get_callers: https://github.com/diffblue/cbmc/blob/develop/unit/analyses/call_graph.cpp#L236-L238
get_reaching_functions: https://github.com/diffblue/cbmc/blob/develop/unit/analyses/call_graph.cpp#L243-L249
etc.

Reply:
Not really, looks like I wrote those too, just a suggestion of a maybe cleaner way of making the same assertion